package.json

```diff
@@ -21,6 +21,7 @@
   "homepage": "https://github.com/coding-berlin/blockstagram#readme",
   "dependencies": {
     "babel-preset-react": "^6.24.1",
+    "babel-preset-stage-2": "^6.24.1",
```

Comment:
what do you need that for?

Reply:
This change was coming from a commit not related to this PR (https://github.com/coding-berlin/blockstagram/pull/28).
The other commit uses the object spread operator, but we settled with just using the specific babel plugin for that.